src/content/changes/highlights/2023-03-23.md

```diff
@@ -0,0 +1,19 @@
+- Grafana Explore: It is a new feature enabled in your Grafana installation. This allows you to execute PromQL (and LogQL later) queries against metrics we collect from your clusters. See [our docs for more information](https://docs.giantswarm.io/getting-started/observability/grafana/access/ https://grafana.com/docs/grafana/latest/explore/)
+- Vertical-Pod-Autoscaler (VPA): New version `v.3.x` is our with new improvements. Highlight that now VPA auto-scales itself. This version requires updating the existing config. Read the [migration docs](https://github.com/giantswarm/vertical-pod-autoscaler-app#moving-from-2xx-to-3xx---breaking-change) and feel free to ask our team in case of any doubt (@support-atlas).
+
+## Apps
+- [external-dns-app](https://github.com/giantswarm/external-dns-app) version [2.34.1](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#2341---2023-03-22) adds `node-role.kubernetes.io/control-plane` to the toleration of crd install jobs. This update also contains the addition of `ServiceMonitor` and the addition of default values that were released in version [2.34.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#2340---2023-03-21)
+- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.14.1](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0141---2023-03-22) updates to Kyverno version 1.9.2 and policy-reporter version 2.17.0. This version includes performance improvements to increase the AdmissionReports processing speed and reduce the number of reports stored in the cluster.
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) version [2.27.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22)This is the second part of our upstream alignement. To be aligned with upstream we added 7 helper properties, 43 deployment properties and added CI values from upstream. Please take care and read the [changelog](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22) as there are some changes that should be done with care. But not only properties got added, in the process of alignement we also changed 9 helpers and 5 deployment properties. Additionally the chart.yaml got changed to be aligned with upstream. Last but not least we removed 2 helper and 9 deployment properties. Please be aware that some of these changes, additons and removals can require some changes in our delpoyments. Please read the [changelog](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22) for all information. 
+- [trivy-app](https://github.com/giantswarm/trivy-app/) version [0.8.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#080---2023-03-17) updates the Trivy scanner version to v0.37.2, containing various bug fixes and additional support for future scanning features.
```

Comment:
```suggestion
- [Kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.14.1](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0141---2023-03-22) updates to Kyverno version 1.9.2 and policy-reporter version 2.17.0. This version includes performance improvements to increase the AdmissionReports processing speed and reduce the number of reports stored in the cluster.
- [Nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) version [2.27.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22)This is the second part of our upstream alignement. To be aligned with upstream we added 7 helper properties, 43 deployment properties and added CI values from upstream. Please take care and read the [changelog](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22) as there are some changes that should be done with care. But not only properties got added, in the process of alignment, we also changed 9 helpers and 5 deployment properties. Additionally, the `chart.yaml` got changed to be aligned with upstream. Last but not least we removed 2 helper and 9 deployment properties. Please be aware that some of these changes, additions and removals can require some changes in our deployments. Please read the [changelog](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22) for all information. 
- [Trivy-app](https://github.com/giantswarm/trivy-app/) version [0.8.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#080---2023-03-17) updates the Trivy scanner version to `v0.37.2`, containing various bug fixes and additional support for future scanning features.
```

Reply:
FWIW I keep the name the actual app name so that people don't copy and break